pages/common/plocate.md

```diff
@@ -0,0 +1,16 @@
+# plocate
+
+> Find filenames quickly.
+> More information: <https://plocate.sesse.net>.
+
+- Look for patterns in the database (recomputed periodically):
+
+`plocate {{pattern}}`
+
+- Look for a file by its exact filename (a pattern containing no globbing characters is interpreted as `*pattern*`):
+
+`locate */{{filename}}`
+
+- Recompute the database (to find recently added files):
+
+`sudo updatedb`
```

Comment:
We shouldn't be documenting other commands in the same page.

Reply:
Agreed, I think we can remove this command from page if necessary.

Reply:
I think updating the database is important because, without it the **plocate** command won't work. Is it possible to add a link to another page in a page?

Reply:
See `tldr fdisk` 

Within the description you can suggest to see other pages.

Reply:
It would take a lot of time to figure out the problem to new users if, they didn't notice that line. Is there any other way?

Reply:
We could maybe add something to the description here such as, `make sure to use the updatedb to include new files`

Reply:
Looks good!